crawl-ref/source/describe.cc

```diff
@@ -2926,6 +2927,26 @@ void describe_spell(spell_type spelled, const monster_info *mon_owner,
     }
 }
 
+/**
+ * Examine a given ability. List its description and details.
+ *
+ * @param ability   The ability in question.
+ */
+void describe_ability(ability_type ability)
+{
+#ifdef USE_TILE_WEB
+    // XXX I have no idea what show_as_menu() does, can't find definition
+    tiles_crt_control show_as_menu(CRT_MENU, "describe_ability");
```

Comment:
`show_as_menu` is not a function at all, it's the name you're giving for your new object of type `tiles_crt_control`. Since you never refer to the object at all, you could just as well call it `asdfkjuke`. Or, better, choose a name that describes what it's for.

Reply:
`show_as_menu` actually does give some clues as to what it does -- it's a RAII thing that tells webtiles to use the menu mode for drawing (as opposed to the plain CRT mode, which is almost but not quite never used any more).